scipy/stats/_multivariate.py

```diff
@@ -1421,8 +1421,9 @@ def mean(self, alpha):
 
         Returns
         -------
-        mu : scalar
-            Mean of the Dirichlet distribution
+        mu : ndarray
```

Comment:
It's both:

    In [48]: stats.dirichlet.mean(alpha=[1])
    Out[48]: 1.0

    In [49]: stats.dirichlet.mean(alpha=[1, 2])
    Out[49]: array([ 0.33333333,  0.66666667])

I suggest making it ``scalar or ndarray``.